### PR TITLE
fr: running msgmerge on fr.po (update from the English main text).

### PR DESCRIPTION
**Pull request by AdrienBaudemont**
fr: running msgmerge on fr.po (update from the English main text).

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -1,7 +1,7 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Comprehensive Rust 🦀\n"
-"POT-Creation-Date: 2023-09-18\n"
+"POT-Creation-Date: \n"
```
Comment (AdrienBaudemont) [inline po/fr.po:4]: My messages.pot file is missing the creation date. Looking into that and will update the PR.

Comment (AdrienBaudemont) [reply po/fr.po]: This should be fixed now.

Comment (mgeisler) [reply po/fr.po]: Yes, it looks good now!

**Comment by mgeisler** [comment]
@sakex and @lb034582341, you're up! :smile:

**Comment by AdrienBaudemont** [comment]
I will update this PR.

**Comment by mgeisler** [comment]
Please remember to run `cargo install mdbook-i18n-helpers` if you haven't done so recently: we made a new release a few weeks ago.

**Comment by AdrienBaudemont** [comment]
> Please remember to run `cargo install mdbook-i18n-helpers` if you haven't done so recently: we made a new release a few weeks ago.

I was already up-to-date, but thank you for pointing this out!

I have updated the PR by merging main into my working branch and rerunning:
```
MDBOOK_OUTPUT='{"xgettext": {"pot-file": "messages.pot"}}'   mdbook build -d po
msgmerge --update po/fr.po po/messages.pot
```

**Review by sakex — APPROVED** [review]
Just skimmed through it, LGTM thanks

**Pull request merged**